package.json

```diff
@@ -11,13 +11,13 @@
     "axe-playwright": "^1.1.11",
     "eslint": "^7.32.0",
     "eslint-config-prettier": "^8.3.0",
-    "prettier": "^2.4.1",
-    "rollup-plugin-import-css": "^2.0.1"
```

Comment:
I'm curious why  rollup-plugin-import-css isn't needed anymore.

Reply:
Some design system components still use this. When they do it is a dependency of the component.

It isn't necessary as a top level dependency in this site build though